client/stylesheets/_modal.scss

```diff
@@ -1136,7 +1136,45 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 /*~~ Join Button End ~~*/
 
 .modJoinWrapper {
+	width: 70% !important;
+	margin: 60px auto;
+}
+
+.modJoinWrapper h1 {
```

Comment:
Merge rule `.modJoinWrapper h1` with rule on line 1138<br>Selector `modJoinWrapper` should be written in lowercase with hyphens